Training 441 Crawl/css/styles.css

```diff
@@ -0,0 +1,84 @@
+/*!
+ * All Rights Reserved
+ * This software is proprietary information of
+ * Intelligent Sense
+ * Use is subject to license terms.
+ * Filename: style.css
+ */
+/*
+ * Author:      fquiros@akurey.com
+ * Date:        30/07/2019
+ * Description: Styles for training 441.
+ */
+
+
+@import url('https://fonts.googleapis.com/css?family=Acme'); 
+
+
+
+.main-container{
+    text-align: center;  
+    margin: 0 auto;
+}
+
+.title{
```

Comment:
I suggest you to install a style lint extension to your IDE. So it tells you to be consistent with spaces and trailing spaces. like `.title {` instead of `.title{`